src/main/java/com/cburch/logisim/file/XmlWriter.java

```diff
@@ -118,13 +118,13 @@ static int stringCompare(String a, String b) {
     final var vb = b.getNodeValue();
     c = stringCompare(va, vb);
     return c;
-        // This can happen in some cases, e.g. two text components
-        // on top of each other. But it seems rare enough to not
-        // worry about, since our normalization here is just for
-        // ease of comparing circ files during testing.
-        // System.out.printf("sorts equal:\n");
-        // System.out.printf(" a: <%s %s>%s\n", na, ma, va);
-        // System.out.printf(" b: <%s %s>%s\n", nb, mb, vb);
+    // This can happen in some cases, e.g. two text components
+    // on top of each other. But it seems rare enough to not
+    // worry about, since our normalization here is just for
+    // ease of comparing circ files during testing.
+    // System.out.printf("sorts equal:\n");
```

Comment:
Despite being commented out this is dead code when uncommented. If you want to keep it for any reason, please ensure `return c;` is the last statement of this method.

Reply:
Agree, will be fixed in the pinbug PR that I'm working on right now. Basically this file has nothing to do in this PR, I just don't know how to get it out :-) Any hints are welcomed

Reply:
Okay, got it done, now the PR is clean